sale_order_pricelist_tax/tests/test_tax.py

```diff
@@ -82,6 +114,79 @@ def test_tax_ttc_fp(self):
         self.assertEqual(sale.amount_total, 12)
         self.assertEqual(sale.amount_untaxed, 10)
 
+    def create_invoice(self, tax1, tax2):
```

Comment:
Can you move all the new test in a separated file? test_price_tax_state.py ?

Then in this file in the setup define the tax and the account to use (so it avoid to do search here)

Reply:
done